src/formats/subrip/subripinputformat.h

```diff
@@ -48,7 +48,7 @@ class SubRipInputFormat : public InputFormat
 
 			offset += m_regExp.matchedLength();
 
-			QString text(data.mid(offset, (unsigned)m_regExp.indexIn(data, offset) - offset));
+			QString text(data.mid(offset, m_regExp.indexIn(data, offset) - offset));
```

Comment:
You're right. This won't work. But the current solution seems also wrong because (2^32 - offset) will be converted back to signed int because indexIn() takes a signed int (that was the reason I removed the cast). I'll fix it later.